### DISCUSSION TITLE
Multiselect pt checkbox classname

### DISCUSSION THREAD

**Discussion by hamyam** [primefaces/community#2209] [PrimeReact]
I'm using Primereact with Tailwind and can't figure out a way to propagate the classname to the `p-checkbox-box`. 

I can access the headers checkbox Box with 
`pt={{ 
        headerCheckbox: {box: {className:"border-2"}},
      }}`

But not the underlying `p-checkbox-box`. I would excpect to have the same passthrough to checkbox. But this refers to `MultiSelectPassThroughType<React.HTMLAttributes<HTMLDivElement>>` instead. 

What am I doing wrong here?

Here's the complete Component: 

```javascript
<MultiSelect  
      value={selectedItems}  
      options={names}  
      onChange={(e: MultiSelectChangeEvent) => {  
        setSelectedItems(e.value);  
        setSelectAll(e.value.length === names.length);  
      }}  
      selectAll={selectAll} 
      onSelectAll={(e: MultiSelectAllEvent) => {
        setSelectedItems(e.checked ? [] : names);
        setSelectAll(!e.checked);
      }}
      virtualScrollerOptions={{itemSize: 43}}
      maxSelectedLabels={3}
      placeholder="Select Items"
      pt={{ 
        headerCheckbox: {box: {className:"border-2"}}
      }}
/>
```

**Comment by MLS535** [comment]
Thanks to you I was able to hide the input checkbox. Have you tried to do something like this? 
`const ptConfig = {
    headerCheckbox: ({ context }) => ({
        input: {
            style: 'display: none;'
        },
        box: {
            style: 'display: none;'
        }
    })
}` And use input to style it. I'm using vue, so maybe is a little bit different.
